src/engine/test/scripts/engine-suite/src/engine_integration/cmds/generate_doc.py

```diff
@@ -0,0 +1,93 @@
+import shared.resource_handler as rs
+
+
+def get_asset_doc_fn(resource_handler: rs.ResourceHandler, results: list):
+    def fn(file_path: str):
+        decoder = resource_handler.load_file(file_path, rs.Format.YML)
+        doc = f'| {decoder["name"]} | {decoder["metadata"]["description"]} |'
+        results.append(doc)
+
+    return fn
+
+
+def run(_, resource_handler: rs.ResourceHandler):
```

Comment:
error?

Reply:
yes it can't find the documentation.yml

```
/wazuh/src/engine/ruleset/wazuh-core
╰─# /usr/local/bin/engine-integration generate-doc   
Traceback (most recent call last):
  File "/usr/local/bin/engine-integration", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/engine_integration/__main__.py", line 34, in main
    args.func(vars(args), resource_handler)
  File "/usr/local/lib/python3.8/dist-packages/engine_integration/cmds/generate_doc.py", line 14, in run
    doc = resource_handler.load_file('documentation.yml', rs.Format.YML)
  File "/usr/local/lib/python3.8/dist-packages/shared/resource_handler.py", line 86, in load_file
    return self._load_file(path, format)
  File "/usr/local/lib/python3.8/dist-packages/shared/resource_handler.py", line 76, in _load_file
    content = path.read_text()
  File "/usr/lib/python3.8/pathlib.py", line 1236, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
  File "/usr/lib/python3.8/pathlib.py", line 1222, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "/usr/lib/python3.8/pathlib.py", line 1078, in _opener
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'documentation.yml'
```

Reply:
This is because the module does not have the documentation yml, there is nothing wrong with the script.